Templates (#9211)
apple/swift-package-manager: Examples/init-templates/README.md

# Swift package templating example

---

> This template project is a simple example of how a template author can create a template and generate a swift projects, utilizing the `swift package init` capability in swift package manager (to come).

Reviewer: ```suggestion
This project provides a simple example to show how to author a template to generate Swift projects, utilizing the template feature included within the `swift package init` command of the Swift Package Manager.
```

## Parts Service

> The parts service template can generate a REST service using Hummingbird (app server), and Fluent (ORM) with configurable database management system (SQLite3, and PostgreSQL). There are various switches to customize your project.

Reviewer: ```suggestion
The parts service template generates a REST service using Hummingbird (app server) and Fluent (ORM) with a configurable database management system (SQLite3 and PostgreSQL). The template includes various switches to customize your project.
```
